modules/nw-dns-operatorloglevel.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+// * networking/dns/nw-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-operatorloglevel_{context}"]
+= Set CoreDNS operatorLogLevel
+
+Cluster Administrators can configure `operatorLogLevel` to more quickly track down OpenShift DNS issues using `logrus`. `logrus` has seven logging levels: Trace, Debug, Info, Warning, Error, Fatal and Panic. After the logging level is set, log entries with that severity or anything above it will be logged. Valid values for operatorLogLevel are `Normal`, `Debug`, and `Trace`. `Trace` has the most detailed information.
```

Comment:
track down OpenShift Cluster DNS Operator which uses logrus.

Reply:
Cluster Administrators can configure `operatorLogLevel` using the `logrus` parameter to more quickly track down OpenShift DNS issues. The `logrus` parameter has seven logging levels: Trace, Debug, Info, Warning, Error, Fatal and Panic.